docs/media-buy/api-reference.md

```diff
@@ -712,12 +714,47 @@ Lists available advertising products for the authenticated principal with option
       "is_custom": false,
       "brief_relevance": "Premium CTV inventory aligns with sports content request and prime time targeting"  // If brief was provided
     }
-  ]
+  ],
+  "policy_compliance": {
+    "status": "approved",
+    "message": "Advertiser and products approved for all inventory"
```

Comment:
or at least the products returned, right? not necessarily all inventory. Do we even need a message if approved? and is "approved" implying too much since there still may be a human approval later